.gitignore

```diff
@@ -8,3 +8,4 @@
 GPATH
 GTAGS
 GRTAGS
+/setenv.bat
```

Comment:
Do we want this? In the long run every contributor might end up with a script like that, but with a different name.

Reply:
Setenv is just my own configuration. I have removed it from gitignore.